docker/config/local_dev.env

```diff
@@ -3,6 +3,7 @@
 # Whether or not this is a local development environment.
 # NOTE(willkg): Do not set this in server environments!
 LOCAL_DEV_ENV=True
+LOGGING_LEVE=INFO
```

Comment:
``LOGGING_LEVE`` → ``LOGGING_LEVEL``

Reply:
The next version of Everett will do what-you-meant configuration. I bet we can throw some AI and ML at it and it'll be epic.